README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](http://calabash-ci.xyz:8081/job/Calabash-iOSDeviceManager/job/develop/badge/icon)](http://calabash-ci.xyz:8081/job/Calabash-iOSDeviceManager/job/develop/)
+
```

Comment:
This link does not work.

I have not found an "embeddable status" URL for the Blue Ocean pipelines.

Reply:
True. This will work when changes will be merged to develop. This will trigger creation of pipeline and image will be available

Reply:
How it looks for current branch:
http://calabash-ci.xyz:8081/job/Calabash-iOSDeviceManager/job/feature%252Fjenkins_pipline/badge/icon